deployments/testnet-deployer-aws.nix

```diff
@@ -1,18 +0,0 @@
-{ ... }:
```

Comment:
I'm worried by the changes introduced to deployers.

The situation is worsened by the fact we do not have a single deployer deployment mechanism between mainnet/staging and testnet.

I'd like to have at least the staging deployer tested:  deploy one before this PR, and then redeploy it with the PR.

Or maybe it's simpler to make sure there's no change at all to the current deployers..

Oh, the temptation for refactoring.. : -/